docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,49 @@
 title: Changelog
 ---
 
+## 12.13.0
+
+_Released 05/23/2023_
+
+**Features:**
+
+- Adds Git-related messages for the
+  [Runs page](https://docs.cypress.io/guides/core-concepts/cypress-app#Runs) and
+  [Debug page](https://docs.cypress.io/guides/cloud/runs#Debug) when users
```

Comment:
```suggestion
  [Runs page](/guides/core-concepts/cypress-app#Runs) and
  [Debug page](/guides/cloud/runs#Debug) when users
```

Reply:
Fixed in https://github.com/cypress-io/cypress-documentation/pull/5272/commits/e482e95287ed1e3ea1cb9a58d91b661d8123da81